tests/lib/managers/folders-test.js

```diff
@@ -276,4 +276,20 @@ describe('Folders', function() {
 		});
 	});
 
+	describe('deletePermanently()', function() {
+
+		it('should call BoxClient defaultResponseHandler method with the callback when response is returned', function(done) {
+			sandbox.mock(boxClientFake).expects('defaultResponseHandler').returns(done);
```

Comment:
@mattwiller how would the result test look like? All other tests of this kind are in this format, but I think there might be a room for improvement. We might discuss it closer over hangouts.